website/pages/en/help.js

```diff
@@ -67,8 +67,8 @@ class Help extends React.Component {
             </header>
             <p>
               <translate>
-                Jest is worked on full-time by Facebook's JavaScript Foundation
-                team. Team members are often around and available for questions.
+                Jest is worked on by a team of volunteers in their spare time.
+                You can find find out ways to talk to other community members below.
```

Comment:
```suggestion
                You can find find out ways to talk community members below.
```

?

even though I personally don't frequent those channels often, I have been there and discussed/answered questions